src/components/atoms/HighlightedPre.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Highlight, { defaultProps } from 'prism-react-renderer';
+import theme from 'prism-react-renderer/themes/dracula';
+import Pre from './Pre';
+
+/* eslint-disable */
+const HighlightedPre = props => {
+  const className = props.children.props.className || '';
+  const matches = className.match(/language-(?<lang>.*)/);
+
+  return (
+    <Highlight
+      {...defaultProps}
+      code={props.children.props.children}
+      language={
+        matches && matches.groups && matches.groups.lang
+          ? matches.groups.lang
+          : ''
+      }
```

Comment:
🤔 Do we have optional chaining and nullish coalescing in this project? If so this can be simplified to:

```suggestion
      language={matches?.groups?.lang ?? ''}
```

Reply:
Looks like we do, nice!